docs/principles/developer-experience.md

```diff
@@ -0,0 +1,62 @@
+# Developer experience principles
+
+## No longer than 1 hour to get up and running on a cold machine 
+
+The average tenure for a developer at the Guardian is low (~2 years). We frequently have new starters who are unfamiliar
```

Comment:
I think my stats were a bit out of date it turns out - we had slightly higher turnover rates when I was a dev manager (correlation but not causation I'm sure! 😬 ). 

But I'm guessing the *median* average is probably still under 2.5 years for devs (excluding test/other roles) although @NataliaLKB can probably correct me.

Reply:
Yes. The most recent figure is actually 3.5 years

Reply:
Wow, that sounds quite good! I'll revise this justification

Reply:
@NataliaLKB is that correct? I.e. is it:

* a median figure
* including people who have left (i.e. not just an average of people here)
* only for developers (i.e. excluding qa's, test, and other roles)

If so then it is a vast improvement over a couple of years ago - possibly because we had a couple of waves of exodus both around dotcom launch and then when austerity hit.

?

Reply:
It is a median figure only for engineers that are not QA only considering people who have left (I believe).